pom.xml

```diff
@@ -320,7 +320,7 @@
     <version.org.jboss.arquillian.extension.drone>2.5.1</version.org.jboss.arquillian.extension.drone>
     <version.org.jboss.arquillian.graphene>2.3.2</version.org.jboss.arquillian.graphene>
     <version.com.google.guava.for-selenium>23.0</version.com.google.guava.for-selenium>
-    <version.org.jboss.errai>4.10.0.Final</version.org.jboss.errai>
+    <version.org.jboss.errai>4.11.0-SNAPSHOT</version.org.jboss.errai>
```

Comment:
@tiagobento the snapshot from ERRAI master is enough or do you need changes from https://github.com/errai/errai/pull/407 ? the CI here will not take the changes from your PR until you merge it and that master head with Snapshot will be deployed by CI. Just to remind it here.

Reply:
@mareknovotny Thanks for the reminder, Marek! We'll probably need to release a new version from Errai though. I'm still validating this solution with other people involved on this upgrade, but I'll keep you updated on the PRs!

Reply:
Errai version upgraded to 4.11.0.Final. https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1488/commits/267c92ed8a9a7c84ca5baf8378b54fb757730b92